client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -105,7 +106,10 @@ const Header: FunctionComponent< Props > = ( { prev } ) => {
 				<div className="gutenboarding__header-group">
 					{ hasSelectedDesign && (
 						<Button
-							href={ `/start/frankenflow?theme=${ selectedDesign?.slug }&siteTitle=${ siteTitle }` }
+							href={ addQueryArgs( '/start/frankenflow', {
+								siteTitle: siteTitle,
+								...( selectedDesign?.slug && { theme: selectedDesign.slug } ),
```

Comment:
With current implementation of signup flow, all args should be provided even if empty. 
So this would be fine:
`/start/frankenflow?theme=&siteTitle=` 
but without args, the signup controller is throwing some error: 
`Error: frankenflow did not provide the query dependencies [theme] it is configured to`